docs/GettingStarted/configuration/ExporterDeploytime.md

```diff
@@ -2,28 +2,71 @@
 The job of the deploy time exporter is to capture the timestamp at which a deployment event happen in a production environment.
 
-In order for proper collection, we require that all deployments associated with a particular application be labelled with the same `app.kubernetes.io/name=<app_name>` label.
+: **NOTE:** In order for proper collection, we require that all deployments associated with a particular application be labelled with the same [APP_LABEL](#app_label), which by default is `app.kubernetes.io/name=<app_name>`.
 
-## Instance Config
+## Example
 
 ```yaml
 exporters:
   instances:
   - app_name: deploytime-exporter
     exporter_type: deploytime
-    env_from_configmaps:
-    - pelorus-config
-    - deploytime-config
+    extraEnv:
+      - name: LOG_LEVEL
+        value: DEBUG
+      - name: NAMESPACES
+        value: my-application1,my-application2
 ```
 
-## ConfigMap Data Values
+## Deploy Time Exporter configuration options
 
-This exporter provides several configuration options, passed via `pelorus-config` and `deploytime-config` variables. User may define own ConfigMaps and pass to the committime exporter in a similar way.
+This is the list of options that can be applied to `env_from_secrets`, `env_from_configmaps` and `extraEnv` section of a Failure time exporter.
 
-| Variable                  | Required | Explanation                                                                                                       | Default Value                 |
-|---------------------------|----------|-------------------------------------------------------------------------------------------------------------------|-------------------------------|
-| `LOG_LEVEL`               | no       | Set the log level. One of `DEBUG`, `INFO`, `WARNING`, `ERROR`                                                     | `INFO`                        |
-| `APP_LABEL`               | no       | Changes the label key used to identify applications                                                               | `app.kubernetes.io/name`      |
-| `PROD_LABEL`              | no       | Changes the label key used to identify namespaces that are considered production environments.                    | unset; matches all namespaces |
-| `NAMESPACES`              | no       | Restricts the set of namespaces from which metrics will be collected. ex: `myapp-ns-dev,otherapp-ci`              | unset; scans all namespaces   |
-| `PELORUS_DEFAULT_KEYWORD` | no       | ConfigMap default keyword. If specified it's used in other data values to indicate "Default Value" should be used | `default`                     |
+| Variable | Required | Default Value |
+|----------|----------|---------------|
+| [LOG_LEVEL](#log_level) | no | `INFO` |
+| [APP_LABEL](#app_label) | no | `app.kubernetes.io/name` |
+| [NAMESPACES](#namespaces) | no | - |
+| [PROD_LABEL](#prod_label) | no | - |
+| [PELORUS_DEFAULT_KEYWORD](#pelorus_default_keyword) | no | `default` |
+
+###### LOG_LEVEL
```

Comment:
I would change these headers all to `###`

Reply:
All our config options across other docs are ###### so why here ### ? Actually I have not checked your PRs for that ?

Reply:
I used `###` because the parent header is `##`